tardis/tests/tests_slow/test_w7.py

```diff
@@ -49,7 +39,8 @@ def setup(self, request, reference, data_path, atomic_data_fname,
         assert self.atom_data.uuid1 == kurucz_data_file_uuid1
 
         # Create a Configuration through yaml file and atom data.
-        tardis_config = Configuration.from_config_dict(config_yaml, self.atom_data)
+        tardis_config = Configuration.from_config_dict(
```

Comment:
@yeganer ... as it didn't work, I took reference from [this](https://github.com/tardis-sn/tardis/pull/543/files#diff-36f090313b4e1f6a024f4cbc0537a305R42) line.